### DISCUSSION TITLE
Expired tokens

### DISCUSSION THREAD

**Discussion by ghost** [community/community#73098] [New to GitHub]
### Select Topic Area

Question

### Body

Im new to Git Hub, so my apologies if this question has already been asked. Once your GitHub token has expired does it delete all the repositories you created? Or do you just have to make a new token to access them? I gave mine an expiration date because that's what it recommended.

**Comment by mdavic0** [comment]
Welcome to GitHub! No need to worry, your repositories won't be deleted when your personal access token expires. When your token expires, you'll simply lose the ability to use it for authentication. To regain access, you'll need to generate a new token with a new expiration date or without an expiration date. Your repositories and their content will remain intact. It's a good security practice to have tokens with expiration dates, but just be sure to refresh or create new ones before they expire to continue using GitHub without any interruptions. If you have any more questions or need further assistance, feel free to ask. Happy coding!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @AnalyticsByShamika ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
